homeassistant/components/dynalite/const.py

```diff
@@ -16,16 +16,43 @@
 CONF_AUTO_DISCOVER = "autodiscover"
 CONF_BRIDGES = "bridges"
 CONF_CHANNEL = "channel"
+CONF_CHANNEL_COVER = "channelcover"
 CONF_CHANNEL_TYPE = "type"
+CONF_CLOSE_PRESET = "close"
 CONF_DEFAULT = "default"
+CONF_DEVICE_CLASS = "class"
+CONF_DURATION = "duration"
 CONF_FADE = "fade"
 CONF_HOST = "host"
 CONF_NAME = "name"
 CONF_NO_DEFAULT = "nodefault"
-CONF_POLLTIMER = "polltimer"
+CONF_OPEN_PRESET = "open"
+CONF_POLL_TIMER = "polltimer"
 CONF_PORT = "port"
 CONF_PRESET = "preset"
+CONF_ROOM = "room"
+CONF_ROOM_OFF = "room_off"
+CONF_ROOM_ON = "room_on"
+CONF_STOP_PRESET = "stop"
+CONF_TEMPLATE = "template"
+CONF_TILT_TIME = "tilt"
+CONF_TIME_COVER = "timecover"
+CONF_TRIGGER = "trigger"
 
 DEFAULT_CHANNEL_TYPE = "light"
+DEFAULT_COVER_CLASS = "shutter"
 DEFAULT_NAME = "dynalite"
 DEFAULT_PORT = 12345
+DEFAULT_TEMPLATES = {
+    CONF_ROOM: [CONF_ROOM_ON, CONF_ROOM_OFF],
```

Comment:
We only prefix the constants with `CONF_` that are config keys not the values. `CONF_ROOM_ON` and `CONF_ROOM_OFF` are values from what I can tell.

Reply:
Looking again, it seems they are keys.

Reply:
opened it again, since it is correct on some of them. CONF_ROOM_ON and CONF_ROOM_OFF are indeed keys and they only appear here so that cv can check that the right keys are provided according to the context of the "template" parameter.
However, a few of them are actually values (e.g. CONF_ACTIVE_INIT). What is the convention for these? should i just change them to DYNALITE_ACTIVE_INIT?

Reply:
Yeah, we don't have a specific convention for values. Just that the constant should have the same name as the string more or less.

Reply:
ok. will change the relevant values to not use CONF

Reply:
done